### DISCUSSION TITLE
Anyhow exaple?

### DISCUSSION THREAD

**Discussion by Je12emy** [poem-web/poem#624] [Q&A] [answered]
Does anyone have a repository showcasing the integration with [anyhow]?

[anyhow]: https://docs.rs/anyhow/latest/anyhow/

**Comment by sunli829** [comment, marked as answer]
When the `anyhow` feature is enabled, `anyhow::Error` can be converted to `poem::Error`, but I don't recommend doing this because it will always be given a `500 InternalServerError` status code.
